documentation/secure_storage.md

```diff
@@ -108,14 +108,13 @@ TEE file is 1024.
 
 Key manager is an component in TEE file system, and is responsible for handling
 data encryption and decryption and also management of the sensitive key
-materials. There are two types of keys used by key manager. One is Secure
-Storage Key (SSK) and another one is File Encryption Key (FEK).
+materials. There are three types of keys used by key manager: the Secure
```

Comment:
I wonder if we should fix the other missing determiner: "...used by the key manager...". Or maybe a separate change to separate it from this change.